examples/synthetic_data/syn_text_pii_scanner_example.ipynb

```diff
@@ -0,0 +1,608 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import torch
</pre><p>remove this line, it is not used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/LeakPro/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>